### PR TITLE
removed gradient and added blur

### PR DESCRIPTION
**Pull request by norachams**

### CODE REVIEW

File: src/components/Navbar/Navbar.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ const Navbar = () => {
     }, [showMenu]);
 
     return (
-        <nav className="gradient fixed top-0 z-50 flex h-fit w-full items-center justify-between p-3 px-10 lg:justify-normal">
+        <nav className="fixed top-0 z-50 flex h-fit w-full items-center justify-between p-3 px-10 lg:justify-normal backdrop-blur-md">
```
Comment (teoh4770) [inline src/components/Navbar/Navbar.tsx:19]: looks great!

**Review by teoh4770 — COMMENTED** [review]
I'm not sure if comingsoon page should include in the navbar

**Comment by teoh4770** [comment]
Screenshot 2024-03-29 at 10 15 27 PM

somehow backdrop blur didn't apply to the sidemenu in this branch

File: src/components/Navbar/Navbar.tsx [hunk 2]
```diff
@@ -28,7 +30,11 @@ const Navbar = () => {
             </div>
 
             <div className="portal-btn hidden transition-all duration-500 ease-in-out lg:mr-32 lg:block">
-                <Button className="xl:px-6 xl:py-3">Application Portal</Button>
+                <Button className="xl:px-6 xl:py-3">
+                    <Link to="./comingsoon">
```
Comment (juancwu) [inline src/components/Navbar/Navbar.tsx:34]:
```suggestion
                    <Link to="/comingsoon">
```

File: src/components/Navbar/Menu.tsx [hunk 3]
```diff
@@ -11,9 +12,8 @@ const Menu: React.FC<MenuProp> = ({ showMenu, hideMenu }) => {
     return (
         <div className="oveflow-hidden fixed right-0 top-0 z-50 h-full w-full max-w-[65%] border px-10 py-24 backdrop-blur-xl">
             <NavItems isHorizontal={false} handleClick={hideMenu} />
-
             <Button className="mt-20 block lg:hidden">
-                Application Portal
+                <Link to="./comingsoon">Application Portal</Link>
```
Comment (juancwu) [inline src/components/Navbar/Menu.tsx:16]:
```suggestion
                <Link to="/comingsoon">Application Portal</Link>
```

**Review by juancwu — CHANGES_REQUESTED** [review]
The mobile

**Review by juancwu — APPROVED** [review]
lgtm

**Comment by aidantrabs** [comment]
> I think the mobile menu bg can be fixed by remove the `h-full` class

this would work but it would have a stopping point for the navbar, not sure how i feel about this. 

thoughts? @juancwu @norachams @teoh4770 
Screenshot 2024-03-30 at 12 27 26 AM

**Comment by teoh4770** [comment]
it's weird XD

File: src/components/Navbar/Menu.tsx [hunk 4]
```diff
@@ -10,7 +10,7 @@ const Menu: React.FC<MenuProp> = ({ showMenu, hideMenu }) => {
     if (!showMenu) return;
 
     return (
-        <div className="oveflow-hidden fixed right-0 top-0 z-50 w-full max-w-[65%] border px-10 py-24 backdrop-blur-xl">
+        <div className="oveflow-hidden gradient fixed right-0 top-0 z-50 h-screen max-w-[65%] border-l border-black px-10 py-24">
```
Comment (juancwu) [inline src/components/Navbar/Menu.tsx:13]: @teoh4770

I don't know how to feel about the menu having a gradient instead

image

Comment (teoh4770) [reply src/components/Navbar/Menu.tsx:13]: Well, you wanna keep the transparent background still?

**Pull request closed without merging**